docs/compose/SingleSelectChipGroup/README.md

```diff
@@ -0,0 +1,39 @@
+# SingleSelectChipGroup
```

Comment:
also here let's have a single readme for both and name it `Chip Group` here (plus update the folder name to `ChipGroup`). make sure to update the package references in the file as well